packages/core/src/indexing/postgres/postgres-index-api.ts

```diff
@@ -36,6 +36,8 @@ export class PostgresIndexApi implements DatabaseIndexApi {
       .insert({
         stream_id: String(args.streamID),
         controller_did: String(args.controller),
+        stream_content: args.streamContent,
+        tip: String(args.tip),
```

Comment:
is casting to a String like this the same as calling `toString()` on the CID object?  I've always used `toString()`, is there an advantage to doing the cast this way instead?

Reply:
I've honestly used it only because we used it initially during the sqlite implementation. It's similar with a major difference though:

`var value = null;
String(value);     // "null" -> String
value.toString(); // TypeError`

I need to look into that `"null"` or `"undefined"` is interpreted by knex correctly as null value vs. doing the check ourselves if we switch to `toString` for nullable fields. 

Overall thats not enough of an advantage though IMHO and I'm OK to switch it to `toString()` instead with a simple in-line `null/undefined` check.